fuse_ll/src/fuse/mount.rs

```diff
@@ -10,21 +11,157 @@ use std::path::Path;
 
 use param::*;
 
+pub struct MountOption {
+    pub name: String,
+    pub parser: fn(&mut FuseMountArgs, &MountOption, &str),
+    pub regex: Regex,
+    #[cfg(target_os = "linux")]
+    pub flag: Option<u64>,
+    #[cfg(target_os = "macos")]
+    pub flag: Option<i32>,
+    #[cfg(target_os = "macos")]
+    pub fuse_flag: Option<u64>,
+}
+
+pub fn get_all_options() -> String {
+    let options: Vec<_> = get_mount_options().iter().map(|x| x.name.clone()).collect();
+    options.join(",")
```

Comment:
merge into one line?

Reply:
Rust cannot infer the type of options, so separate to two lines.

Reply:
let a = [1, 2, 3];

let doubled = a.iter().map(|x| x * 2).collect::<Vec<i32>>();

Reply:
Thanks